core/src/main/java/bisq/core/api/CoreTradesService.java

```diff
@@ -273,15 +289,33 @@ Trade getTrade(String tradeId) {
                 ));
     }
 
+    List<Trade> getOpenTrades() {
+        coreWalletsService.verifyWalletsAreAvailable();
+        coreWalletsService.verifyEncryptedWalletIsUnlocked();
+        return tradeManager.getTrades();
+    }
+
+    List<TradeModel> getTradeHistory(GetTradesRequest.Category category) {
+        coreWalletsService.verifyWalletsAreAvailable();
+        coreWalletsService.verifyEncryptedWalletIsUnlocked();
+        if (category.equals(CLOSED)) {
+            var closedTrades = closedTradableManager.getClosedTrades().stream()
+                    .map(t -> (TradeModel) t)
+                    .collect(Collectors.toList());
+            closedTrades.addAll(bsqSwapTradeManager.getBsqSwapTrades());
```

Comment:
Shouldn't we use `getConfirmedBsqSwapTrades` here?

Reply:
A bsq swap offer becomes an executed trade right after it is taken.  It should show up in the trade history with 0 tx confirmations until the next block, but the swap has been made.

Reply:
If I call `getConfirmedBsqSwapTrades` here, before the first confirmation, `gettrades --category=closed` will not display the executed trade.